Index: vorbis.pc.in
================================================================================
--- vorbis.pc.in
+++ vorbis.pc.in
@@ -8,7 +8,8 @@
 Name: vorbis
 Description: vorbis is the primary Ogg Vorbis library
 Version: @VERSION@
-Requires: ogg
+Requires.private: ogg
 Conflicts:
-Libs: -L${libdir} -lvorbis -lm
+Libs: -L${libdir} -lvorbis 
+Libs.private: -lm
 Cflags: -I${includedir}
--- vorbisenc.pc.in
+++ vorbisenc.pc.in
@@ -8,7 +8,7 @@
 Name: vorbisenc
 Description: vorbisenc is a library that provides a convenient API for setting up an encoding environment using libvorbis
 Version: @VERSION@
-Requires: vorbis
+Requires.private: vorbis
 Conflicts:
 Libs: -L${libdir} -lvorbisenc
 Cflags: -I${includedir}
--- vorbisfile.pc.in
+++ vorbisfile.pc.in
@@ -8,7 +8,7 @@
 Name: vorbisfile
 Description: vorbisfile is a library that provides a convenient high-level API for decoding and basic manipulation of all Vorbis I audio streams
 Version: @VERSION@
-Requires: vorbis
+Requires.private: vorbis
 Conflicts:
 Libs: -L${libdir} -lvorbisfile
 Cflags: -I${includedir}
